explainer.md

```diff
@@ -0,0 +1,369 @@
+  - [Are SPC credentials identical to WebAuthn credentials?](#are-spc-credentials-identical-to-webauthn-credentials)
+  - [Why use the PaymentRequest API](#why-use-the-paymentrequest-api)
+- [Alternatives Considered](#alternatives-considered)
+  - [Traditional WebAuthn](#traditional-webauthn)
+  - [Delegated Authentication](#delegated-authentication)
+- [Security Considerations](#security-considerations)
+  - [Enrollment in cross-origin iframes](#enrollment-in-cross-origin-iframes)
+  - [Cross-origin authentication ceremony](#cross-origin-authentication-ceremony)
+  - [Merchant-supplied data](#merchant-supplied-data)
+- [Privacy Considerations](#privacy-considerations)
+  - [Probing](#probing)
+  - [Credential-sharing](#credential-sharing)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Motivation
+
+Online payments are usually a 3 party interaction:
+
+* The Merchant (e.g. an online web store)
+* The Customer (e.g. the user buying from an online web store)
+* The ??? (e.g. the bank that issued the payment instrument being used)
+
+> **NOTE**: It is not uncommon for there to be a fourth party: the Payment
+> Service Provider, whom the Merchant may delegate the task of accepting the
+> payment method to. This is usually done by the Merchant either redirecting the
+> User to the Payment Service Provider's website, or by embedding the Payment
+> Service Provider in an iframe. Any following mention of the term Merchant
+> should be taken to mean either a Merchant or a Payment Service Provider;
+> whichever is handling payment.
+
+Traditionally, the Customer only interacts with the Merchant, and the Merchant
+communicates with the ??? via a back-channel protocol. However, ???s are
+increasingly looking to authenticate Customers during online payments. This is
+done for both regulatory reasons (e.g. [SCA] in the European Union) or for
+fraud prevention reasons - either fradulent Customers or fradulent Merchants.
+
+Existing methods of authenticating a Customer during an online payment are
+either high friction (e.g. embedding challenge iframes from the ???) or have
+poor user privacy (e.g. fingerprinting or tracking the user for risk analysis,
+to provide a 'frictionless flow'). The payments industry needs a consistent,
+low friction, strong authentication flow for online payments.
+
+The [Web Authentication (WebAuthn) API][webauthn] makes FIDO-based
+authentication available on the web, which provides a strong, low-friction
+method for a user to prove a pre-established identity with a given site.
+However, WebAuthn is not immediately suitable to solve the payments
+authentication problem, as:
+
+1. It requires the ??? to be present in the transaction (e.g. via an embedded
+   iframe), which increases user friction and lowers conversion rates.
+1. The generated assertion contains no payments-related information, which could
+   be useful to meet e.g. [Dynamic Linking] requirements.
```

Comment:
The generated assertion contains no payments-related information, so it cannot be used as-is to fulfill regulatory requirements to provide evidence of user consent (cf. [Dynamic Linking] requirements).